src/app/_components/_sharedcomponents/CardArea/CardArea.tsx

```diff
@@ -3,23 +3,25 @@ import { Box } from "@mui/material";
 import GameCard from "../Cards/GameCard/GameCard";
 import { ICardAreaProps } from "@/app/_components/Gameboard/GameboardTypes";
 
-const CardArea: React.FC<ICardAreaProps> = ({ cards }) => {
+const CardArea: React.FC<ICardAreaProps> = ({ cards, pile }) => {
```

Comment:
I think we can get rid of this wrapper all together. It's not providing a lot of utility and by having components directly using the `GameCard` compnent they can better control the layout of the cards as well as which size and elements a cards should display.